api-dummy/.docker/nginx-server.conf

```diff
@@ -0,0 +1,27 @@
+server {
+    server_name localhost;
+    listen 80;
+    root /srv/app/public;
+
+    location / {
+        try_files $uri /index.php$is_args$args;
+    }
+
+    location ~ ^/index\.php(/|$) {
+        fastcgi_pass api-dummy-fpm:9000;
+        fastcgi_split_path_info ^(.+\.php)(/.*)$;
+        include fastcgi_params;
+
+        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
+        fastcgi_param DOCUMENT_ROOT $realpath_root;
+
+        internal;
+    }
+
+    location ~ \.php$ {
+        return 404;
+    }
+
+    error_log /var/log/nginx/app_error.log;
```

Comment:
in the default image logs probably go to stderr, which makes `docker logs` collect them; may be unnecessary to override these two

Reply:
so I'd remove the two `error_log` and `access_log` directives